src/web/browser/dynamicImport/init.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-new-func */
+/* eslint-disable no-new */
+
+import dynamicImportPolyfill from 'dynamic-import-polyfill';
+import { startup } from '@root/src/web/browser/startup';
+
+// Provides an __import__ function to use for dynamic imports. **Only works on
+// browsers that cut the mustard (support modules).**
+export const initialiseDynamicImport = () => {
+    try {
+        new Function('url', `return import(url)`);
+        self.__import__ = (url: string) =>
```

Comment:
What's the benefit of `self` over `window`? It seems like it might be not be required in this context given we're unlikely to run this in a worker?